src/shared/components/oncoprint/controls/OncoprintControls.tsx

```diff
@@ -207,6 +207,7 @@ export default class OncoprintControls extends React.Component<
     @observable heatmapGenesReady = false;
     @observable private _selectedTreatmentIds: string[] = [];
     private textareaTreatmentText = '';
+    @observable filteredTreatmentOptions = this.props.treatmentSelectOptions;
```

Comment:
its problematic to initialize a variable with the value of props, since the initial props might not be meaningful, it might be an empty array for example.